documentation/features/plugins/configure-notificaties-api-plugin.md

```
@@ -29,6 +29,26 @@ An example plugin configuration:&#x20;
 
 After saving the configuration Valtimo automatically ensures the required kanalen exist and that an abonnement with a generated secret is registered against the configured Notificaties API instance. Disabling this behaviour is possible by setting `valtimo.zgw.register-abonnementen=false` in the `application.yml`.
 
+## Available actions
+
+The Notificaties API plugin supports the following actions that can be configured in process links.
+
+A general description on how to create process links can be found [here](configure-plugin.md).
```

markdownlint-cli2 (0.22.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)